### PR TITLE
feat: remove edundant code

### PR DESCRIPTION
**Pull request by zqran**

### CODE REVIEW

**Comment by darkskygit** [comment]
target will be undefined when an invalid profile is incoming externally

File: apps/web/next.config.mjs [hunk 1]
```diff
@@ -23,7 +23,7 @@ const profileTarget = {
 };
 
 const getRedirectConfig = profile => {
-  const target = profileTarget[profile || 'dev'] || profileTarget['dev'];
+  const target = profileTarget[profile || 'dev'];
```
Comment (zqran) [inline apps/web/next.config.mjs:26]:
```ts
const target = profileTarget[profile || 'dev'] || profileTarget['dev'];
```

when profile is `undefined` will fallback to `'dev'`, `profileTarget[profile || 'dev']` has been achieved.

Comment (darkskygit) [reply apps/web/next.config.mjs:26]: if we pass profile like `abcd`, at this point target is `undefined`

Comment (zqran) [reply apps/web/next.config.mjs:26]: oh, got it~

**Pull request closed without merging**